### ISSUE TITLE
Accessibility criterion

### ISSUE THREAD

**Issue by deftransform** [pebtrocious/discordless#13]
The current comparison lack a way to determine what kind of accessibility features exist in the various tools. I'm no really knowledgeable about that to contribute but the information would be welcome.
